catalog_validation/utils.py

```diff
@@ -11,6 +11,7 @@
     'show_subquestions_if': object,
     'type': str,
     'editable': bool,
+    'hide': bool,
```

Comment:
`hidden` perhaps? to mimic html? :P

Reply:
Or is that helm chart default?

Reply:
Sure :p And it's not related to helm at all :)